org.eclipse.emfcloud.validation/src/main/java/org/eclipse/emfcloud/validation/ValidationFramework.java

```diff
@@ -186,8 +183,23 @@ public void updateRecentValidationResult(List<ValidationResult> validationResult
 		return result;
 	}
 
-	private boolean isDiagnosisMessage(int code, String source) {
-		if (code == 0 && source == "org.eclipse.emf.ecore") {
+	private boolean isErrorDiagnostic(JsonNode responseData) {
+		if (isDiagnosisMessage(responseData) || isSeverityOK(responseData)) {
```

Comment:
Shouldn't this be: 
`if (!isDiagnosisMessage(responseData) || isSeverityOK(responseData)) {`
So if it is not a diagnosis or a diagnosis but ok then it is not an error, or am I missing something?

Reply:
The `isErrorDiagnostic()` method returns false (so the data gets filtered) if one of the if statements is 'true' and is `isDiagnosisMessage()` returns true, if the data is the default message.

So it should be right this way (also tested by the tests).

Reply:
ok, got it makes sense